packages/flutter/test/material/app_test.dart

```diff
@@ -435,4 +435,27 @@ void main() {
     // Default Cupertino US "select all" text.
     expect(find.text('Select All'), findsOneWidget);
   });
+
+  testWidgets('Temp fake window test', (WidgetTester tester) async {
+    final WidgetsBinding binding = TestWidgetsFlutterBinding.ensureInitialized();
+    assert(binding is TestWidgetsFlutterBinding);
+    final TestWidgetsFlutterBinding testBinding = binding;
+    testBinding.window.textScaleFactorTestValue = 20.0;
+
+    double textScaleFactorInMaterialApp;
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Builder(
+          builder: (BuildContext context) {
+            textScaleFactorInMaterialApp = MediaQuery.of(context).textScaleFactor;
+
```

Comment:
extraneous blank line

Reply:
(it separates the first have of the pumpWidget call and the second half of the call into two separate paragraphs, which is weird given that it's one statement)